### DISCUSSION TITLE
Machine Learning

### DISCUSSION THREAD

**Discussion by carlossolis27** [pybricks/support#671] [Q&A]
Hi everyone 

Just a quick question im really enjoying coding with pybrick but im wondering if the new feature machine learning that lego will add for mindstorm inventor would be possible in pybricks ? or its already possible somehow ? im not an expert im just an amateur student asking silly questions

**Comment by repkovsky** [comment]
Here you can find something you were looking for https://github.com/orgs/pybricks/discussions/1007.
